scripts/kconfig-split.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+# This is a slightly modified version of ChromiumOS' splitconfig
+# https://chromium.googlesource.com/chromiumos/third_party/kernel/+/stabilize-5899.B-chromeos-3.14/chromeos/scripts/splitconfig
```

Comment:
@tych0 and others. I will update #635 either tonight or tomorrow. I have some more thoughts which I would like us to discuss